docs/api.md

```diff
@@ -523,8 +515,8 @@ const main = async () => {
   init(basePath, port);
   await emulator.start(port);
 
-  const blockOffset = await getBlockOffset();
-  console.log({ blockOffset });
+  const [blockOffset, err] = await getBlockOffset();
+  console.log(blockOffset, err);
```

Comment:
We can wrap args in curly braces for better readability :)

Reply:
Changed into the following format: `console.log({ blockOffset }, { err })`. Please let us know if this isn't the format you are looking for.